command.go

```diff
@@ -298,18 +240,17 @@ func (c Command) startApp(ctx *Context) error {
 	app.Version = ctx.App.Version
 	app.HideVersion = ctx.App.HideVersion
 	app.Compiled = ctx.App.Compiled
-	app.Author = ctx.App.Author
-	app.Email = ctx.App.Email
 	app.Writer = ctx.App.Writer
 	app.ErrWriter = ctx.App.ErrWriter
+	app.ExitErrHandler = ctx.App.ExitErrHandler
 	app.UseShortOptionHandling = ctx.App.UseShortOptionHandling
 
-	app.categories = CommandCategories{}
+	app.Categories = newCommandCategories()
```

Comment:
❓ _(blocking question)_ This is another `Categories` change that I'm not sure belongs in V2?

Reply:
This has been updated